episodes/mri.md

```diff
@@ -328,7 +329,15 @@ print(type(t2_data))
 numpy.ndarray
 ```
 
-The data is a multidimensional **array**.
+The data is a multidimensional **array**. Of note we can also use the array by accessing the loaded file's dataobj property:
+
+```python
+t2_img.dataobj
```

Comment:
I did not commit this because , I want them to see how it returns the object to spark a discussion as it doesn't show the valuesm\, but shows the array-like object